Makefile

```diff
@@ -497,20 +497,30 @@ webhook-certs-dir:
 _ensure-kong-system-namespace:
 	@kubectl create ns kong-system 2>/dev/null || true
 
+TMP_DIR=$(shell pwd)/tmp
```

Comment:
Why not use `mktemp`? This will pollute the working dir and won't be cleaned after e.g. host restart.